java/swagger-petstore/src/main/webapp/index.html

```diff
@@ -0,0 +1,61 @@
+<!-- HTML for static distribution bundle build -->
```

Comment:
use always latest version of ui, as done in other 2.0 samples, eg. java-jersey2-configfile or others, this means keep only updated index.html and modify pom to download ui

Reply:
Done!